### PR TITLE
Add release email

### PR DESCRIPTION
**Pull request by tildedash**
Fix https://3.basecamp.com/3178566/buckets/33458756/todos/6386876800

### CODE REVIEW

File: bin/release_email.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  local,
+  eventmakerLocalEndpoint
+} = argv;
+
+const endpoint = local ? eventmakerLocalEndpoint : API_ENDPOINT;
+const apiClient = new ApiClient(endpoint, token);
+
+const replaceProductionURL = (releaseDir, theme) => {
+  const host = `${PRODUCTION_ENDPOINT}/${releaseDir}`;
+  const reg = new RegExp(`https://${EMAIL_FAKE_HOST}`, "g");
```
Comment (robinmonjo) [inline bin/release_email.js:66]: I don't think we need fake host replacement here do we ? On websites, this is used to make the assets URL (js and css) ok, but on email I think it's not needed

File: bin/release_email.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+const deleteReleaseDir = () => {
+  fs.rmSync(EMAIL_BUCKET_ROOT_DIR, { recursive: true, force: true });
+}
+
+logInfo(`Starting upload for ${theme}`);
+const releaseDir = EMAIL_BUCKET_ROOT_DIR;
+const templateDir = `email_themes`;
+coypEmailTheme(theme, releaseDir, templateDir, () => {
```
Comment (robinmonjo) [inline bin/release_email.js:88]: If we indeed do not need to replace anything in the email theme and just upload it as is, we can remove this step and just send the template folder to s3

**Review by robinmonjo — CHANGES_REQUESTED** [review]

File: bin/release_email.js (hunk 1 above)
Comment (tildedash) [reply bin/release_email.js:66]: I use it to have the same nomenclature in the specs.yml

Comment (robinmonjo) [reply bin/release_email.js:66]: ah yes indeed you need it for this !! Ok let's keep it that way then 😊

**Review by robinmonjo — APPROVED** [review]

**Pull request merged**